### DISCUSSION TITLE
API key unlock error 'Invalid Master Password'

### DISCUSSION THREAD

**Discussion by alelucual18** [dani-garcia/vaultwarden#4728] [Q&A]
I dont know what happens I try to use api cli and I exec bw  login with user and pass everything ok but when use bw login --apikey send client_id and secretId perfect but when exec bw unlock the master password always is Invalid always, the same master password that use in bw login with user and pass. Some help please

**Comment by BlackDex** [comment]
See the closed issue: #4720
